article/app/views/liveblog/liveBlogBodyContentAMP.scala.html

```diff
@@ -28,6 +29,11 @@
         @if(model.currentPage.currentPage.isArchivePage) {
             @fragments.liveBlogNavigation(article.content.metadata.id, model.currentPage, amp = true)
         }
+
+            <script type="application/ld+json">
+            @Html(LiveBlogPosting(model).toString())
```

Comment:
It is not AMP only, so I think I made a mistake here. I originally thought it would be, but that was before I saw how we'd implemented the microdata. It looks like I missed the template @NataliaLKB mentioned above so I should probably move it there. Will double check!